### DISCUSSION TITLE
how to change the font and the font size

### DISCUSSION THREAD

**Discussion by GmosNM** [helix-editor/helix#5984] [Q&A] [answered]
help

**Comment by gabydd** [comment, marked as answer]
You need to change that in your terminal emulator, there should be some form of documentation for the terminal emulator (which you are running helix in) that will specify how to configure the font.

(If you are still having trouble just let us know which terminal emulator you are using so we can help)

**Reply by GmosNM** [reply]
before i zoom in
![image](https://user-images.githubusercontent.com/37900384/218760718-cc191bf5-f043-4f4e-8bcb-715422d412bc.png)
after i zoom in 
![image](https://user-images.githubusercontent.com/37900384/218760831-05a50ca8-fa6a-486b-b877-a62b9e8861e3.png)
i can't see the ui

[this is the normal cmd for windows]
but i tried it with Nushell and its good thank you <3

**Reply by quomat** [reply]
On Windows you need to right on the status bar (where you see "Command Prompt - hx") and click "Properties..."
I recommend Windows Terminal and Powershell though, using command prompt got me a few graphical glitches.

**Reply by ecotrissy** [reply]
I really want to be able to zoom/text size using CTRL ++ / CTRL +- / CTRL + Wheel.
TBH this is a bit of deal breaker for me.

**Reply by quomat** [reply]
@ecotrissy You can do it in Windows Terminal

**Reply by ecotrissy** [reply]
Sorry, yes, I see now.
